### PR TITLE
Add ROC curve, change fontsize

### PR DESCRIPTION
**Pull request by mikael10j**

### CODE REVIEW

**Comment by mikael10j** [comment]
The roc curve is not correct

File: gammaboard.py [hunk 1]
```diff
@@ -4,10 +4,12 @@
 
 import ctaplot
 import numpy as np
+import matplotlib
```
Comment (vuillaut) [inline gammaboard.py]: not used?

File: gammaboard.py [hunk 2]
```diff
@@ -429,11 +448,13 @@ def create_resolution_fig(site='south', ref=None):
...
         fig, axes
     """
-    fig, axes = plt.subplots(2, 2, figsize=(12, 8))
+    fig, axes = plt.subplots(3, 2, figsize=(12, 12))
     ax_ang_res = axes[0][0]
     ax_ene_res = axes[0][1]
     ax_imp_res = axes[1][0]
     ax_eff_area = axes[1][1]
+    ax_roc = axes[2][0]
+    # ax_auc = axes[2][1]
```
Comment (vuillaut) [inline gammaboard.py]: why is this commented?

File: gammaboard.py [hunk 3]
```diff
@@ -448,8 +469,22 @@ def create_resolution_fig(site='south', ref=None):
...
         ax_eff_area.legend()
 
+    ax_roc.plot([0, 1], [0, 1], linestyle='--', color='r', alpha=.5)
+    ax_roc.set_xlim([-0.05, 1.05])
+    ax_roc.set_ylim([-0.05, 1.05])
+    ax_roc.set_xlabel('False Positive Rate')
+    ax_roc.set_ylabel('True Positive Rate')
+    ax_roc.set_title('Receiver Operating Characteristic')
+
+    # ax_auc.set_axis_off()
```
Comment (vuillaut) [inline gammaboard.py]: commented ?

**Review by vuillaut — CHANGES_REQUESTED** [review]

File: gammaboard.py (hunk 1 above)
Comment (mikael10j) [reply gammaboard.py]: removed

File: gammaboard.py (hunk 2 above)
Comment (mikael10j) [reply gammaboard.py]: sorry didn't push the commit

**Review by vuillaut — APPROVED** [review]

**Pull request merged**